app/lib/postcodes.js

```diff
@@ -36,11 +36,11 @@ function lookup(res, next) {
           next();
           break;
         case 404:
-          debugPostcodes({ url, response: postcodeRes.statusCode });
+          log.warn({ res: postcodeRes }, '404 from postcodes.io');
```

Comment:
The failing postcode can be seen in the subsequent debug line of the log (http://localhost:3000/finders/results?location=*HG5+0JK*&open=true&context=) but it might be easier when scanning the log file to have both the URL and status code in the message.